README.md

```diff
@@ -40,6 +40,7 @@ To add a dependency using Maven:
     * [scalardb](https://javadoc.io/doc/com.scalar-labs/scalardb/latest/index.html) - A library that makes non-ACID distributed databases/storages ACID-compliant
     * [scalardb-rpc](https://javadoc.io/doc/com.scalar-labs/scalardb-rpc/latest/index.html) - Scalar DB RPC libraries
     * [scalardb-server](https://javadoc.io/doc/com.scalar-labs/scalardb-server/latest/index.html) - Scalar DB Server that is the gRPC interfarce of Scalar DB
+    * [scalardb-schema-loader](https://javadoc.io/doc/com.scalar-labs/scalardb-schema-loader/latest/index.html) - Scalar DB Schema Loader that is using for creating and deleting the schema in Scalar DB
```

Comment:
```suggestion
    * [scalardb-schema-loader](https://javadoc.io/doc/com.scalar-labs/scalardb-schema-loader/latest/index.html) - Scalar DB Schema Loader that creates and deletes Scalar DB schemas 
```

Can you also change the following page as well?
https://github.com/scalar-labs/scalardb/blob/80746946236c0a8c435a5c40cd9117c1a8f59a6e/docs/index.md

Reply:
Thanks for the suggestions. Updated in [commit](https://github.com/scalar-labs/scalardb/pull/413/commits/1e9d9976b52b9a88be55d01b7c3caf36b3c4f811)